quantum/wpm.h

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2018 Jack Humbert <jack.humb@gmail.com>
```

Comment:
You may want to change this

Reply:
Yes indeed!  Thank you @mechmerlin for spotting this.